configurations/tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
     "compileOnSave": false,
     "compilerOptions": {
-      "lib": ["ES2015"],
+      "lib": ["ES2015", "DOM"],
```

Comment:
This might lead to Node code depending on DOM APIs and Typescript not alerting us about it. I'd remove it and solve the issue that you tried to solve here differently.